Documentation/components/drivers/thermal/index.rst

```diff
@@ -0,0 +1,157 @@
+=================
+Thermal Framework
+=================
+
```

Comment:
Please include an introduction to let people know what is the Thermal Framework, what is its purpose, etc

Reply:
Done